CHANGELOG.md

```diff
@@ -4,13 +4,45 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Common Changelog](https://common-changelog.org/).
 
+[1.2.0]: https://github.com/sablier-labs/v2-core/compare/v1.1.2...v1.2.0
 [1.1.2]: https://github.com/sablier-labs/v2-core/compare/v1.1.1...v1.1.2
 [1.1.1]: https://github.com/sablier-labs/v2-core/compare/v1.1.0...v1.1.1
 [1.1.0]: https://github.com/sablier-labs/v2-core/compare/v1.0.2...v1.1.0
 [1.0.2]: https://github.com/sablier-labs/v2-core/compare/v1.0.1...v1.0.2
 [1.0.1]: https://github.com/sablier-labs/v2-core/compare/v1.0.0...v1.0.1
 [1.0.0]: https://github.com/sablier-labs/v2-core/releases/tag/v1.0.0
 
+## [1.2.0] - 2024-07-01
+
+### Changed
+
+- **Breaking:** Move common logic into `Lockup` contract ([#784](https://github.com/sablier-labs/v2-core/pull/784),
+  [#813](https://github.com/sablier-labs/v2-core/pull/813), [#850](https://github.com/sablier-labs/v2-core/pull/850),
+  [#941](https://github.com/sablier-labs/v2-core/pull/941))
+- Disallow zero `startTime` ([#813](https://github.com/sablier-labs/v2-core/pull/813),
+  [#852](https://github.com/sablier-labs/v2-core/pull/852))
+- Change permissions of `withdraw` function to public ([#785](https://github.com/sablier-labs/v2-core/pull/785))
+- Replace `ISablierV2Recipient` with `ISablierLockupRecipient` hook interface
+  ([#951](https://github.com/sablier-labs/v2-core/pull/951))
+- Remove `try..catch` block from hook callbacks ([#951](https://github.com/sablier-labs/v2-core/pull/951))
+- Only allow assets with supported characters in symbol ([#960](https://github.com/sablier-labs/v2-core/pull/960),
```

Comment:
saying "allow" before "only" would be better,

also, should we also mention that this is about  NFT Descriptor?

"Allow only supported characters in NFT Descriptor asset symbols"

Reply:
Yes I agree with you.